libs/ui-lib/lib/ocm/hooks/_use-feature-detection.test.tsx

```diff
@@ -27,7 +27,7 @@ const makeSpyOn$OrganizationsApi$getOrganization = (mockResponse: Organization)
 describe('use-feature-detection.ts', () => {
   it('Sets only external features when no overrides are provided', async () => {
     const DummyComponent: React.FC = () => {
-      useFeatureDetection();
+      useFeatureDetection(STANDALONE_DEPLOYMENT_ENABLED_FEATURES);
```

Comment:
This test-case is checking what happens when no arguments are passed to the hook. Please revert this change and if necessary add an additional test-case.

Reply:
changes done